paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -156,6 +157,18 @@ def add_subparser(subparsers):
             "Default is %(default)s seconds."
         ),
     )
+    list_parser.add_argument(
+        "-w",
+        "--warn",
+        dest="warn",
+        type=int,
+        default=DEFAULT_WARN_PERCENT,
+        help=(
+            "Percent of timeout to warn at if the deployment hasn't finished. "
+            "For example, --warn=75 will warn at 75%% of the timeout. "
+            "Defaults to %(default)s."
```

Comment:
took me a second to realize that the `s` here is part of argparse's format string